### DISCUSSION TITLE
rdt-client windows 11 and a mapped shared drive

### DISCUSSION THREAD

**Discussion by Kyrunner** [rogerfar/rdt-client#582] [General]
Does rdt-cliet not support mapped share drive in Windows 11? I have rdt-client and radarr running on Windows 11 everything works fine as long as I set the rdt-client to download to my Windows machine the file will download and radarr will move it to my qnap share drive. I'm trying to eliminate the download to the Windows box and downloading it strait to the qnap drive would save some time but whenever I try to add the qnap share drive to rdt-clit for downloads it fails and says it can't see the share drive.

**Comment by Ravioli1G** [comment]
Have you tried mounting it as a network drive rather than a network share before the rtdc.exe starts? If the network drive was mounted after rtdc starts you'll have to restart it again.
